metadata-ingestion/src/datahub/ingestion/source/state/checkpoint.py

```diff
@@ -213,7 +213,7 @@ def to_checkpoint_aspect(
                 ),
             )
             checkpoint_aspect = DatahubIngestionCheckpointClass(
-                timestampMillis=int(datetime.utcnow().timestamp() * 1000),
+                timestampMillis=int(datetime.now().timestamp() * 1000),
```

Comment:
```suggestion
                timestampMillis=int(datetime.now(tz=timezone.utc).timestamp() * 1000),
```

same change below

in general, utcnow() should never be used

we should avoid using datetime.now() without a timezone, but `.now().timestamp()` technically produces the right result for the wrong reasons

Reply:
I didn't add the tz because we're just getting the timestamp, but I'm fine adding if we want to be consistent / prevent future errors